overlay.js

```diff
@@ -0,0 +1,333 @@
+document.addEventListener('keydown', documentEvent => {
+    if (documentEvent.shiftKey && documentEvent.key === 'L' && !document.querySelector('#skater-overlay')) {
+        // const browser = chrome || browser;
+        createOverlay();
+        setUpInputEventListener();
+        setTimeout(() => focusInput(), 100);
+    } else if (getSearchInputElement()) {
+        focusInput();
+        const focusedElement = document.querySelector('.focused');
+        switch(documentEvent.key) {
+            case "Up":
+            case "ArrowUp":
+                // move to last search result or input
+                // FIXME functionize
+                if (focusedElement.getAttribute('class') === 'skater-link skater-result-0 focused') {
+                    focusInput();
+                    // FIXME: implement preventDefault better
+                    documentEvent.preventDefault();
```

Comment:
probably want this moved up to as high as it can be. likely all the way to the top